<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="save_close_pushButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SaveMatrix</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>591</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_matrix">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>551</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Size matrix</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_matrix"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="AcceptSize">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>92</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>AcceptSize</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="radok_tablewidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>551</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Line b1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveLine">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SaveLine</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Result">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Random_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Randomly Fill</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="res_tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>551</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>290</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
